.circleci/config.yml

```diff
@@ -55,6 +87,70 @@ jobs:
             ./bin/linux/amd64/github-release release -t "${CIRCLE_TAG}"
             cat release-description.md | grep ${PROJECT} | ./bin/linux/amd64/github-release edit -t ${CIRCLE_TAG} -d -
 
+  py3to2:
+    docker:
+      - image: python:3.7.0-slim
+    steps:
+      - checkout
+      # Install some requirements
+      - run: pip install pytest future strip-hints future-fstrings[rewrite]
+
+      # Rename files and paths to `rest` and make strings py2 compatible
+      - run: find . -exec dirname {} \; | grep "aio" | sed 's/aio/rest/g' | xargs -L1 mkdir -p
+      - run: for file in $(find . -type f | grep "aio"); do cp -r $file $(echo "$file" | sed 's/aio/rest/g'); done
+      # For some files with conflicts due to unicode encoding in py2, do not use
```

Comment:
The comment is probably misleading and it is about `future-fstring-show` redifining strings and bytes. Some examples of places where this is an issue are:
- https://github.com/talkiq/gcloud-aio/blob/aa2436759b6cd926cdb405f9e46d7ea38c832a3e/datastore/tests/unit/value_test.py#L18
- https://github.com/talkiq/gcloud-aio/blob/aa2436759b6cd926cdb405f9e46d7ea38c832a3e/datastore/tests/unit/value_test.py#L13 (hence the need for the `if...else` in https://github.com/talkiq/gcloud-aio/blob/c06802930b12de9485daa0e5649827d6a40bab4e/datastore/tests/unit/value_test.py#L15

Most of these issues are in `datastore` package because we determine the message type based on the variable's `type` and that gets modified with the class redefinitions.